docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3'
```

Comment:
If you guys are feeling up to it, you can also deploy this in your k3s cluster running via rancher desktop rather than using docker-compose, since that will emulate this running in kubernetes, but on your desktop environment.

Reply:
Then you can do your local dev against it just like using docker-compose.